backend/setup_pyodide.mjs

```diff
@@ -0,0 +1,17 @@
+// setup_pyodide.ts
+import { loadPyodide } from "pyodide";
+async function run() {
+    const pyodide = await loadPyodide();
+
+    await pyodide.loadPackage("micropip");
+
+    const micropip = pyodide.pyimport("micropip");
+
+    await micropip.install([
+    "typing-extensions",
```

Comment:
You can remove it...it is a dependency of pydantic so it will be installed automatically https://github.com/pydantic/pydantic/blob/dac3c437870a056839c0a485ae3112c76d5e4a7f/pyproject.toml#L47